src/res_parsetree_viewer.ml

```diff
@@ -167,8 +167,15 @@ let funExpr expr =
          (((Labelled _ | Optional _) as lbl), defaultExpr, pattern, returnExpr);
      pexp_attributes = attrs;
     } ->
-      let parameter = Parameter {attrs; lbl; defaultExpr; pat = pattern} in
-      collect attrsBefore (parameter :: acc) returnExpr
+      (* Normally attributes are attached to the labelled argument, e.g. (@foo ~x) => ...
+         In the case of `@res.async`, pass the attribute to the outside *)
```

Comment:
Ah, there was a reason! I'm going to close my #676

Reply:
How about adding tests for this case? Maybe,
```rescript
let f = async (@attr1 ~x, @attr2 ~y) => x + y
```

Reply:
Good point.